### DISCUSSION TITLE
Failed to create 1.1.1 stack

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#4854] [Hubs Cloud Questions] [answered]
Hello,

I have a 1.1.1 custom client wich I made some modifications early this year. 

I want to use it on another project, but I can't create a 1.1.1 stack. AWS gives me a nodejs10 error.

Using 1.1.2 works perfectly. I tried to deploy my custom client over it, but it gives a blank page after I deploy it.

Is it possible to use 1.1.1 or it's obsolete?

Thanks

**Comment by pattersonbl2** [comment, marked as answer]
That version is obsolete.  The runtimes for node and python have to be updated to 14. x and 3.6 respectively.
